packages/mobile/src/transactions/TransactionReview.tsx

```diff
@@ -55,12 +95,13 @@ class TransactionReview extends React.PureComponent<Props> {
     return confirmationProps
   }
 
-  renderCard = (type: TokenTransactionType, confirmationProps: any) => {
+  renderCard = (type: TokenTransactionType, props: any) => {
```

Comment:
small nit i missed - could we type these props?

Reply:
done. also cleaned up those methods we dont need